ansible-runner/context/app/project/roles/mirror_appliance/tasks/migrate-pre-1_3_11.yaml

```diff
@@ -0,0 +1,13 @@
+- name: Create Postgres Password Secret
+  containers.podman.podman_secret:
+    state: present
+    name: pgdb_pass
```

Comment:
nit: do we want to replace it with `{{ pgdb_password }}` defined in [secret-vars.yaml](https://github.com/quay/mirror-registry/blob/main/ansible-runner/context/app/project/roles/mirror_appliance/tasks/secret-vars.yaml#L5)